### ISSUE TITLE
Qubes R4.2.0-rc2 Qubes OS Global Config tool not see qubes-u2f installed in sys-usb

### ISSUE THREAD

**Issue by Szewcson** [QubesOS/qubes-issues#8463]
### Qubes OS release
4.2.0-rc2 after in-place upgrade

### Brief summary
Qubes OS Global Config tool report that "The Qubes U2F Proxy service is not installed in the USB Qube"

### Steps to reproduce
1.) Create sys-usb based on fedora-38 minimal with  qubes-usb-proxy, qubes-input-proxy-sender and qubes-u2f that fallback to qubes-ctap installed.
2.) Install qubes-u2f-dom0 which fallback to qubes-ctap-dom0 in dom0
3.) Enable qubes-u2f-proxy service in all qubes which should have access to u2f
4.) Open Qubes OS Global Config tool
5.) Try to setup u2f rules using that tool

### Expected behavior
sys-usb should be avaliable to use with u2f-proxy

### Actual behavior
Qubes OS Global Config tool report that "The Qubes U2F Proxy service is not installed in the USB Qube". After prompting sys-usb to USB Qube field, error message not disapear but it is possible to check some options and assign qubes. After clicking apply everything selected disappears.

**Comment by Szewcson** [comment]
I want to check it on debian which should have still qubes-u2f but it is uninstallable on both debian-11 and debian-12:
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 qubes-u2f : Depends: python3-u2flib-host but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by Szewcson** [comment]
I have some old debian-11 based template that I just upgraded from Qubes 4.1 and it has qubes-u2f installed. When there is qubes-u2f instead of qubes-ctap Qubes OS Global Config tool sees that and allow changing settings of u2f proxy.

**Comment by Kenny606** [comment]
@marmarek 
The update in testing did indeed fix my non functional u2f proxy functionality and the global config tool on a clean install of RC3 I performed last weekend.
And while you probably already know this, the following comes up during an update, reinstall or clean install of the new version on any of my debian-12 tenplates. 
```
Setting up qubes-ctap (2.0.4+deb12u1) ...
/usr/bin/deb-systemd-helper: error: unable to read qubes-ctapproxy@sys-usb.service
qubes-ctapproxy.service is a disabled or a static unit not running, not starting it.

```
Despite that though as I said it's gotten everything fully functional so seems to be just a minor annoyance. But at first it did give the impression it wasn't going to work. Eventually I just decided to stop tinkering and test a VM anyway and it was working fine so like I said just a side note. Maybe it's somehow just specific to me, maybe something I did troubleshooting before I stumbled on this issue.

Thanks,

Kenny
